services/Skyra.Grpc/Startup.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Skyra.Grpc.Services;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Skyra.Database;
+
+namespace Skyra.Grpc
+{
+	public class Startup
+	{
+		// This method gets called by the runtime. Use this method to add services to the container.
+		// For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+		public void ConfigureServices(IServiceCollection services)
+		{
+			services.AddGrpc();
+			services.AddSingleton<SkyraDbContext>();
```

Comment:
This should be scoped - not singleton, each request should have a new dbcontext otherwise there will be out of date issues & race conditions

```suggestion
			services.AddScoped<SkyraDbContext>();
```